feature/one-more-file.txt

```diff
@@ -0,0 +1,3 @@
+This file was also added into feature1 branch
```

Comment:
Its really good that you have decided to add this line of code here

Reply:
Thanks for this comment Biplab